changelog.txt

```diff
@@ -1,3 +1,17 @@
+4.6.0
+============
+New Features:
+**MSAL .NET now stores the application token returned from the iOS broker (Authenticator)**. This may result in the user experiencing less prompts. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1396)
+**New TokenCacheNotificationArgs.IsApplicationCache property simplifies the development of token cache serialization**. TokenCacheNotificationArgs now include a flag named `IsApplicationCache`, which disambiguates between the app token cache and the user token cache. 
+
+Bug Fixes:
+- **Device Code Flow would fail with a misleading error message if the app was misconfigured in the Azure Application Portal**. MSAL.NET now provides a better error message. - #1407
+- **Setting a non tenanted authority when calling AcquireTokenXX is now ignored**. #1456 
+- **Setting an authority audience of `AzureADMyOrg` and a tenant ID would fail**. It's now possible to specify `.WithAuthority(audience)` and `.WithTenantId()` #1320 
+ 
+Fundamentals:
+- Added tests which check cache format interoperability between MSAL Java and MSAL .NET
+
```

Comment:
Do we want to add a link to the blog post? https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/msal-net-4.6 ?

Reply:
@jmprieur good idea. we don't always have the blog post ready before the release :) thanks.